prov/efa/test/efa_unit_tests.c

```diff
@@ -32,6 +35,10 @@ int main(void)
 		cmocka_unit_test_setup_teardown(test_rdm_cq_read_failed_poll, reset_global_mocks, NULL),
 		cmocka_unit_test_setup_teardown(test_rdm_cq_read_bad_send_status, reset_global_mocks, NULL),
 		cmocka_unit_test_setup_teardown(test_rdm_cq_read_bad_recv_status, reset_global_mocks, NULL),
+#if HAVE_EFADV_CQ_EX
+		cmocka_unit_test_setup_teardown(test_rdm_cq_read_recover_forgotten_peer_ah, reset_global_mocks, NULL),
```

Comment:
better always have this test, but do not do anything if `HAVE_EFADV_CQ_EX` is 0.

Reply:
That is odd. The test will pass if we don't do anything, which means even if we are not handling unknown AH, the test still says it is working.

I think it should be conditional on edfadv capability.

Reply:
The other test should say `skipped` instead of `pass`.